tsconfig.json

```diff
@@ -1,8 +1,27 @@
 {
-  "extends": "./tsconfig.base",
   "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "downlevelIteration": true,
+    "jsx": "react",
+    "lib": ["DOM", "ES2015", "ES6", "DOM.Iterable", "ScriptHost"],
+    "moduleResolution": "Node",
+    "skipLibCheck": true,
+    "noEmit": true,
+    "preserveConstEnums": true,
+    "sourceMap": true,
+    "strict": true,
     "module": "ES6",
     "resolveJsonModule": true,
-    "esModuleInterop": true
-  }
+    "esModuleInterop": true,
```

Comment:
Previously `tsconfig.base` was being extended into `tsconfig.json` and `tsconfig.commonjs.json`

To my eye the commonJS config is also dead code, and we can therefore consolidate the configs?

Also, there is no compile target, so the `target` field can be removed if `module` is provided.